common/scripts/ci/sdk_examples_stop.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+#
+# This script builds and releases just the proxy to heroku. The proxy can be configured to work against any
```

Comment:
The comment is probably not right, please update it to reflect what this thing really does.